source/components/atoms/Checkbox/Checkbox.tsx

```diff
@@ -19,48 +29,24 @@ const CheckboxTick = styled(Icon)`
   margin-top: -3px;
 `;
 
-// TODO: THEME/STYLING MOVE sizes to theme declaration in theme.js
-const sizes = {
-  small: {
-    width: 18,
-    height: 18,
-    padding: 0.5,
-    margin: 4,
-    borderWidth: 2,
-    borderRadius: 3,
-  },
-  medium: {
-    width: 35,
-    height: 35,
-    padding: 0.5,
-    margin: 4,
-    borderWidth: 2,
-    borderRadius: 7,
-  },
-  large: {
-    width: 52,
-    height: 52,
-    padding: 0,
-    margin: 4,
-    borderWidth: 3.2,
-    borderRadius: 10,
-  },
-};
-
-const iconSizes = {
-  small: 18,
-  medium: 36,
-  large: 50,
-};
+interface Props {
+  checked?: boolean;
+  onChange: () => void;
+  disabled?: boolean;
+  size?: 'small' | 'medium' | 'large';
+  color: string;
+}
 
-const Checkbox = props => {
-  const { checked, onChange, size, disabled, color } = props;
+const Checkbox: React.FC<Props> = ({checked, onChange, size, disabled, color}) => {
   const theme = useContext(ThemeContext);
-  const style = {
-    ...sizes[size],
-  };
-  const validColorSchema = getValidColorSchema(color);
-  const tickIcon = <CheckboxTick size={iconSizes[size]} name="done" />;
+
+  // some type trickery to make typescript happy
+  let validColorSchema: 'blue' | 'green' | 'red' | 'purple'; 
```

Comment:
This could be moved to a stand alone type since it's used in several components at this stage :)

Reply:
Even in several places within this component :)

Reply:
Yeah, agreed, I was thinking the same thing but not sure where to put it. I think I'll make a separate typescript file in the styles folder that can export this type, as well as a typed version of the getValidColorSchema function.

Reply:
Yeah that feels like a good idea (y)

Since that's the origin of the type :D

Reply:
I have now done exactly that, added a themeHelpers.ts which contains two color types and a typed getValidColorSchema function.